src/libraries/System.Text.Encodings.Web/src/System/Text/Encodings/Web/SpanUtility.cs

```diff
@@ -23,237 +23,5 @@ public static bool IsValidIndex<T>(Span<T> span, int index)
         {
             return ((uint)index < (uint)span.Length) ? true : false;
         }
```

Comment:
Is it worth it to keep these? I am seeing it is simple check to do it. Or this has perf impact?

Reply:
Removed